sql/src/test/java/io/crate/protocols/postgres/ConnectionContextTest.java

```diff
@@ -51,6 +52,19 @@
     private SQLOperations sqlOperations;
     private List<SQLOperations.Session> sessions = new ArrayList<>();
 
+    private final Authentication authService = new Authentication() {
```

Comment:
Would it make sense to have this as kind of default implementation? Would eliminate the special null handling in a few places